index.html

```diff
@@ -0,0 +1,104 @@
+<!-- Declare document type as HTML -->
+<!DOCTYPE html>
+<!-- Start HTML document -->
+<html lang="en">
+
+<head>
+    <!-- Declare character set -->
+    <meta charset="UTF-8" />
+    <!-- Declare compatibility with Internet Explorer -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <!-- Declare viewport configuration -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Set title of the webpage -->
+    <title>My Personal Portfolio</title>
+    <!-- Link to external stylesheet -->
+    <link rel="stylesheet" href="css/styles.css" />
+    <!-- Include Fontawesome icons kit -->
+    <script src="https://kit.fontawesome.com/7f35089b8a.js" crossorigin="anonymous"></script>
+    <!-- Include Google Fonts link -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet" />
+</head>
+
+<body>
+    <!-- Start header section -->
+    <header>
```

Comment:
- [ ] Kindly ensure that the background image doesn't get cut off by applying it in the `header` container's tag, so it covers the whole `header`, like it is in the design, so as to ensure a pixel-perfect design.

- Your design
<img width="375" alt="wrong" src="https://user-images.githubusercontent.com/41163571/228055941-f31096ef-5da0-4de8-ab1a-003039361acf.png">

- Figma design
<img width="236" alt="right" src="https://user-images.githubusercontent.com/41163571/228056013-322e1f2c-93c1-4f1b-b7e0-46a3b6a945c5.png">

Reply:
Hello, I dont really understand what you mean. Could you kindly elaborate further?